data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,117 @@ mission "Strider notification"
+	on complete
+		payment 8000
+		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should head back to the spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`
+
+
+
+mission "Station Exploration 2"
+	name "Station Exploration"
+	description "Bring a group of Hai and human classmates back home to <planet>."
+	minor
+	source "Darkrest"
+	destination "Greenwater"
+	passengers 5
+	to offer
+		has "Station Exploration 1: done"
+	on offer
+		conversation
+			`The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`
+				accept
+	on visit
+		dialog `You look for your passengers, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		conversation
+			`Upon landing, you are accosted by local police.`
+			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
+			choice
+				`	"Am I in trouble?"`
+				`	"I'm innocent!"`
+			`	"We just want to let you know what happened," he says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
+			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. They're not in trouble, as there's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
+			choice
+				`	"So, am I in trouble or not?"`
+					goto trouble
+				`	"Are my passengers going to be okay?"`
+			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`
+			label trouble
+				`"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
```

Comment:
Idk why but in my head they were younger, probably bc you used classmates instead of students in the description?